metrics/network/network-metrics-iperf3.sh

```diff
@@ -0,0 +1,540 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test measures the following network essentials:
+# - bandwith simplex
+# - bandwith duplex
+# - jitter
+#
+# These metrics/results will be got from the interconnection between
+# a client and a server using iperf tool.
+# The following cases are covered:
+#
+# case 1:
+#  container-server <----> container-client
+#
+# case 2"
+#  container-server <----> host-client
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/lib/network-common.bash"
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+# Port number where the server will run
+port="${port:-5201}"
+fwd_port="${fwd_port:-$port:$port}"
+# Image name
+image="${image:-gabyct/network}"
```

Comment:
Can we translate this into the local Dockerfile idiom please, and use the funcs you added like https://github.com/kata-containers/tests/blob/master/metrics/lib/common.bash#L98, so we hold the Dockerfile locally in the repo :-)

Reply:
@grahamwhaley, currently iperf3 is having a lot of issues in the installation https://github.com/esnet/iperf/issues/153 that is the main reason I am not using a Dockerfile. After this issue is solved, I will change the script to build the Dockerfile as we currently we have in the other metrics.

Reply:
@grahamwhaley , the `iperf_dockerfile` is using `iperf2` which currently is not broken. And the way to reproduce the issue run a container and then download the latest `iperf3` and then try to install it (first you will get an error of libiperf.so that does not exist https://github.com/esnet/iperf/issues/168). Now to solve this issue what I did was the suggested workaround (install lib32z1) and it worked, however, if you want to perform a `iperf3 --version` is broken which is related with the first issue  esnet/iperf#153.

Reply:
Ah, and, iirc we need to build from source as we need a newer-than-distro-provided iperf3 in order to get the TMPDIR support to work around a 9p issue...
That's just crazy that iperf3 still has bugs open from 2014...
OK, if I had some time I'd have a fiddle, but not right now...